scripts/lint_auxiliary_window_close_shortcuts.py

```
@@ -0,0 +1,123 @@
+    except FileNotFoundError:
+        raise ValueError(f"missing {OWNER_LIST_PATH}") from None
+
+    marker = f"private let {OWNER_LIST_NAME}"
+    marker_index = text.find(marker)
+    if marker_index < 0:
+        raise ValueError(f"missing {OWNER_LIST_NAME} in {OWNER_LIST_PATH}")
+
+    list_start = text.find("[", marker_index)
+    list_end = text.find("]", list_start)
+    if list_start < 0 or list_end < 0:
+        raise ValueError(f"could not parse {OWNER_LIST_NAME} in {OWNER_LIST_PATH}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)